### PR TITLE
LingitAS.Lingdys4 version 4.5.111.0

### PR DESCRIPTION
**Pull request by Ogobor**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/78539)

### CODE REVIEW

File: manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+License: None
+LicenseUrl: https://lingit.no/
+Copyright: https://lingit.no/
+CopyrightUrl: https://lingit.no/
+ShortDescription: LingDys4
+Description: https://lingit.no/
+Moniker: https://lingit.no/
+Tags:
+- LingDys
+ReleaseNotes: https://lingit.no/
```
Comment (Trenly) [inline manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml:23]: A URL is not release notes.

File: manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ReleaseNotesUrl: https://lingit.no/
```
Comment (Trenly) [inline manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml:24]: Please ensure this is a direct link to the release notes, not just to the package homepage

File: manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+PurchaseUrl: https://lingit.no/
+InstallationNotes: LingDys installed.
```
Comment (Trenly) [inline manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml:26]:
```suggestion
```
Not needed

File: manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+PublisherUrl: https://lingit.no/
+PublisherSupportUrl: https://lingit.no/
+PrivacyUrl: https://lingit.no/
+Author: LingIT
+PackageName: Lingdys 4
+PackageUrl: https://lingit.no/
+License: None
+LicenseUrl: https://lingit.no/
+Copyright: https://lingit.no/
+CopyrightUrl: https://lingit.no/
```
Comment (Trenly) [inline manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml:17]: Please ensure this is a direct link to the copyright, not just to the package homepage

File: manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+PackageLocale: nb-NO
+Publisher: Lingit AS
+PublisherUrl: https://lingit.no/
+PublisherSupportUrl: https://lingit.no/
+PrivacyUrl: https://lingit.no/
+Author: LingIT
+PackageName: Lingdys 4
+PackageUrl: https://lingit.no/
+License: None
+LicenseUrl: https://lingit.no/
```
Comment (Trenly) [inline manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml:15]: Please ensure this is a direct link to the license, not just to the package homepage

File: manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+PackageVersion: 4.5.111.0
+PackageLocale: nb-NO
+Publisher: Lingit AS
+PublisherUrl: https://lingit.no/
+PublisherSupportUrl: https://lingit.no/
+PrivacyUrl: https://lingit.no/
+Author: LingIT
+PackageName: Lingdys 4
+PackageUrl: https://lingit.no/
+License: None
```
Comment (Trenly) [inline manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml:14]: A URL isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

File: manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml [hunk 7]
```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: LingitAS.Lingdys4
+PackageVersion: 4.5.111.0
+PackageLocale: nb-NO
+Publisher: Lingit AS
+PublisherUrl: https://lingit.no/
+PublisherSupportUrl: https://lingit.no/
+PrivacyUrl: https://lingit.no/
```
Comment (Trenly) [inline manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml:10]: Please ensure this is a direct link to the privacy policy, not just to the package homepage

File: manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml [hunk 8]
```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: LingitAS.Lingdys4
+PackageVersion: 4.5.111.0
+PackageLocale: nb-NO
+Publisher: Lingit AS
+PublisherUrl: https://lingit.no/
+PublisherSupportUrl: https://lingit.no/
```
Comment (Trenly) [inline manifests/l/LingitAS/Lingdys4/4.5.111.0/LingitAS.Lingdys4.locale.nb-NO.yaml:9]: Please ensure this is a direct link to the publisher support page, not just to the package homepage

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**